README.md

```diff
@@ -41,3 +41,26 @@ Connect to Zipkin ui
 ```
 http://localhost:9411/zipkin/
 ```
+
+### K8S
+Setup cluster:
+```
+https://docs.dapr.io/operations/hosting/kubernetes/cluster/
+```
+I used minikube option:
+minikube config set vm-driver virtualbox
+minikube start --cpus=4 --memory=4096 --no-vtx-check --embed-certs - it is suggested to use it without those 2 flags but they were necesarry for me
+
+Make sure you are connected to minikube cluster:
+kubectl config get-contexts
+
+dapr init -k --dev (I had to unlock k8s in Docker Desktop)
+
+kubectl apply -f k8s/server.yaml
+
+wait for pods to be created
+
+kubectl describe pod <server-pod-name>
+Check IP address of a pod
+
+kubectl exec -it <server-pod-name> -- curl <ip-address>:8001/health
```

Comment:
code in codespaces will be more readable. Also description of command I sometimes before and sometimes after command - unification recommended.